source/extensions/filters/http/lua/lua_filter.h

```diff
@@ -10,6 +10,8 @@
 #include "source/extensions/filters/http/common/factory_base.h"
 #include "source/extensions/filters/http/lua/wrappers.h"
 
+#include "wrappers.h"
```

Comment:
ditto.

Reply:
Is this needed? I think this duplicated above. Either delete or use fully qualified paths.